### DISCUSSION TITLE
Can I use Firestore to make cheap browser games with cache persistence only?

### DISCUSSION THREAD

**Discussion by marcusx2** [firebase/firebase-js-sdk#7752] [Q&A] [answered]
I'd like to know if it's possible to use Firestore to make those [cheap browser games](https://armorgames.com) where the user's save game only persists until the user clears his browser's cache. So I'd like to use Firestore to persist offline and between browser sessions, but only until the cache is cleared. There would be no need to actually store anything in Firebase.

I guess I would need to configure [offline persistence](https://firebase.google.com/docs/firestore/manage-data/enable-offline#configure_offline_persistence)? What's the best configuration for this use case? The data doesn't have to ever be stored to Firebase.

**Comment by wu-hui** [comment, marked as answer]
You would need to use `PersistentCacheSettings` *and* `disableNetwork()`. `disableNetwork()` needs to be called at the beginning of your app, and stay unchanged throughout. The SDK will attempt to send the local writes to server whenever `enableNetwork()` is called (which is the default when neither `disableNetwork` and `enableNetwork` is called.)
